contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -59,6 +60,32 @@ specifically, a test is eligible for promotion to conformance if:
   are robust
 - it has a name that is a literal string
 
+### Base Profile Requirements
+
+The Base profile contains only conformance tests for GA, non-optional features or APIs. More
+specifically, a test is eligible for the Base profile if:
+
+- it tests only GA, non-optional features or APIs (e.g., no alpha or beta
+  endpoints, no feature flags required, no deprecated features)
+- it is non-privileged (e.g., does not require root on nodes, access to raw
+  network interfaces, or cluster admin permissions)
```

Comment:
"cluster admin permissions" does not seem very well defined... you can grant specific permission to the APIs a test makes use of without granting full cluster-admin access

Reply:
true. the intent is to capture those things that may affect the cluster as a whole and those things that may affect the node as a whole. As Derek put it in the meeting today "root on cluster" and "root on node". We want to separate out those things that have broad impact across workloads as opposed to those things that an individual "ordinary" workload relies upon.